GoodDay/lecture_12/UniqueItem.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace lecture_12
+{
+    class UniqueItem
+    {
+        static int staticId;
+        public int id;
```

Comment:
it's ok but should be readonly Property such as
`public int Id {get; private set;}`